tools/springboot/write_manifest.sh

```diff
@@ -35,7 +35,6 @@ JAVA_VERSION=$(echo "$JAVA_STRING" | head -n1 | cut -d ' ' -f 3 | rev | cut -c2-
 echo "Manifest-Version: 1.0" > $MANIFESTFILE
 echo "Created-By: Bazel" >> $MANIFESTFILE
 echo "Built-By: Bazel" >> $MANIFESTFILE
-echo "Throw-Down: hootenanny" >> $MANIFESTFILE
```

Comment:
Sorry for this old farce. I put it in on day one of the rule a few years ago as a placeholder for the real manifest lines, and it just kept on keeping on.